webpack.config.ts

```diff
@@ -412,18 +447,18 @@ let appConfig = {
 };
 
 if (IS_TEST || IS_ACCEPTANCE_TEST || IS_STORYBOOK) {
-  appConfig.resolve.alias['integration-docs-platforms'] = path.join(
+  appConfig.resolve!.alias!['integration-docs-platforms'] = path.join(
```

Comment:
A little dirty, but `webpack.Configuration` of course allows these to be undefined. We know we set these above so we just have the explicit null assertion here.